### DISCUSSION TITLE
How to get remote address header in request?

### DISCUSSION THREAD

**Discussion by shivshankardayal** [ntex-rs/ntex#318] [Q&A] [answered]
I am trying to get remote address of a request but when I dump the request there is no such header. How can I find it?

**Comment by fafhrd91** [comment, marked as answer]
usually “host” header contains remote addr. 

you can get real remote ip addr via https://docs.rs/ntex/latest/ntex/web/struct.HttpRequest.html#method.peer_addr
but that could be address of proxy

**Reply by shivshankardayal** [reply]
@fafhrd91 Thanks for the help. `host` is containing the servers IP:Port, however, `peer_address` does contain the remote IP and port.
